frontend/case/src/app/lesson/lesson-edit.component.ts

```diff
@@ -8,7 +8,7 @@ import { Lesson } from './lesson.entity';
 })
 export class LessonEditComponent implements OnInit {
 
-  @Input() lesson: Lesson;
+  @Input() currentLesson: Lesson;
```

Comment:
Why the prefix (current)? Will this component hold more than one?

Reply:
I got confused as lesson also named the lesson in the parent item.
Accordingly, I should change the name in the parent since they hold multiple lessons.
the edit only one.